client/landing/subscriptions/styles.scss

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
This stylesheet duplicates a lot of what is in the stepper global stylesheet. Can we abstract and reuse instead of duplicating?

Reply:
I do not see a suitable place where I could put this code in at the moment. I do not think that it would be a good idea for me to place it in either the `landing/stepper`, or `landing/subscription-manager`, and then to import by the other one, because it would be too easy for someone not to realize the stylesheet may be used by another entry point when making changes to it. 

If you do not mind, I will leave those 20 lines of duplicated `.scss` for the time being of this PR, and I will follow up with another PR to resolve this duplication. I am really eager to get this one in, because the parallel ongoing work keeps on creating conflicts.

Reply:
I created the following task for the above: https://github.com/Automattic/wp-calypso/issues/74888